httpapi/v2/documents_api.go

```diff
@@ -72,3 +79,66 @@ func (h handler) CreateDocument(w http.ResponseWriter, r *http.Request) {
 	render.Status(r, http.StatusCreated)
 	render.JSON(w, r, resp)
 }
+
+// Update updates a pending document.
+// @summary Updates a pending document.
+// @description Updates a pending document.
+// @id update_document_v2
+// @tags Documents
+// @accept json
+// @param authorization header string true "Hex encoded centrifuge ID of the account for the intended API action"
+// @param body body v2.UpdateDocumentRequest true "Document Update request"
+// @param document_id path string true "Document Identifier"
+// @produce json
+// @Failure 400 {object} httputils.HTTPError
+// @Failure 404 {object} httputils.HTTPError
+// @Failure 500 {object} httputils.HTTPError
+// @Failure 403 {object} httputils.HTTPError
```

Comment:
Do we need error code 403 here? Looks like this API doesnt produce this error?

Reply:
it will if the auth is not passed